fastembed/common/onnx_model.py

```diff
@@ -41,6 +41,9 @@ def load_onnx_model(self, model_dir: Path, threads: Optional[int], max_length: i
         onnx_providers = ["CPUExecutionProvider"]
 
         so = ort.SessionOptions()
+        if os.getenv("SLURM_JOB_ID") is not None:
+            so.intra_op_num_threads = int(os.getenv("SLURM_CPUS_ON_NODE"))
+            so.inter_op_num_threads = int(os.getenv("SLURM_CPUS_ON_NODE"))
```

Comment:
Are these values guaranteed to exist if `SLURM_JOB_ID` is set? If not, it'll crash every time like https://github.com/qdrant/fastembed/actions/runs/8688810469/job/23825304587.